recipe/meta.yaml

```
@@ -24,6 +24,10 @@ requirements:
     - netcdf4
     - xarray >=0.12.0
```

Need to bump up minimum xarray version here.

```suggestion
    - xarray >=0.17.0
```

Also, I see in https://github.com/intake/intake-xarray/pull/101 that Python 3.6 is not in CI anymore, does that mean we should bump up minimum Python version to >=3.7 too?

---

Aha!

---

Yes, should be 3.7 now I think